dlt/common/schema/utils.py

```diff
@@ -900,19 +900,41 @@ def get_root_table(tables: TSchemaTables, table_name: str) -> TTableSchema:
     return table
 
 
-def get_nested_tables(tables: TSchemaTables, table_name: str) -> List[TTableSchema]:
+def get_nested_tables(
```

Comment:
do we have unit tests for that function? you should add unit tests for new params.